firebase-database/src/androidTest/java/com/google/firebase/database/QueryTest.java

```diff
@@ -3054,6 +3661,45 @@ public void onCancelled(DatabaseError error) {}
     IntegrationTestHelpers.waitFor(done);
   }
 
+  @Test
+  public void integerKeysBehaveNumericallyStartAfter1()
```

Comment:
Another strange suffix. If these are used to distinguish test cases, then we should come up with more specific names.